pkg/modules/authorization_test.go

```diff
@@ -685,7 +686,7 @@ func TestAuthorizationIngress(t *testing.T) {
 
 			return true, true, tmpCR, sourceClient
 		},
-		"success - creating v1.10.0": func(*testing.T) (bool, bool, csmv1.ContainerStorageModule, ctrlClient.Client) {
+		"success - creating v2.0.0-alpha": func(*testing.T) (bool, bool, csmv1.ContainerStorageModule, ctrlClient.Client) {
```

Comment:
The scenario "success - creating" already tests the v2.0.0-alpha version. This scenarios is testing with the wrong custom resource. It should be using` testdata/cr_auth_proxy_v1100.yaml`

Reply:
Oops, you're right, thank you. Didn't notice the previous test case. Fixed

Reply:
Can you update the CR for the "success - creating v1.10.0" scenario to `testdata/cr_auth_proxy_v1100.yaml`?

Reply:
Done